contents/news/2023-06-storing-secrets.md

```diff
@@ -21,11 +21,11 @@ Environment variables are a common way to store application credentials outside
 
 ## Limit access to credentials
 
-Grant access to application credentials only to the right people (least privilege principe).
+Grant access to application credentials only to the right people (least privilege principle).
 
 ## Use secrets management solutions
 
-Secrets management solutions, such as AWS Secrets Manager, Terraform Vault, provide advanced functionality to securely store and manage application credentials. They provide mechanisms for encryption, auditing, and automatic secrets rotation. By using these solutions, you can centralize the credentials management and reduce risks related to their storage.
+Secrets management solutions, such as AWS Secrets Manager, Terraform Vault, provide advanced functionalities to securely store and manage application credentials. They provide mechanisms for encryption, auditing, and automatic secrets rotation. By using these solutions, you can centralize the credential management and reduce risks related to their storage.
```

Comment:
```suggestion
Secrets management solutions, such as AWS Secrets Manager, Terraform Vault, provide advanced functionalities to securely store and manage application credentials. They provide mechanisms for encryption, auditing, and automatic secrets rotation. By using these solutions, you can centralize the credentials management and reduce risks related to their storage.
```

I think this should stay as crendentials since there isn't only one 😉

Reply:
done